.github/workflows/ci.yml

```diff
@@ -341,7 +341,7 @@ jobs:
             -e PYTHONUNBUFFERED=1 \
             -e SNUBA_SETTINGS=docker \
             -e DEBUG=1 \
-            -e DEFAULT_BROKERS=kafka:9092 \
+            -e DEFAULT_BROKERS=kafka:9093 \
```

Comment:
was this necessary?

Reply:
PR isn't done, but sadly yes. It was needed to fix the sentry -> snuba tests (because of how kafka advertises open ports in devservices, which is how we run Kafka for sentry -> snuba tests in CI now)